imblearn/pipeline.py

```diff
@@ -167,6 +167,15 @@ def _validate_steps(self):
                 % (estimator, type(estimator))
             )
 
+    def _iter(
```

Comment:
At first I thought that we might need tests for this but since the tests of the pipeline are passing might be just a sanity check. So it's ok.